Main/terraform.tfstate

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
state file shouldn't be checked in .. you can remove this from here

Reply:
okay I will remove